Jenkinsfile

```diff
@@ -759,6 +759,25 @@ pipeline {
                         }
                     }
                 } // stage('Functional on CentOS 8')
+                stage('Functional on CentOS 7 with Valgrind') {
+                    when {
+                        beforeAgent true
+                        expression { ! skipStage() }
+                    }
+                    agent {
+                        label 'ci_vm9'
+                    }
+                    steps {
+                        functionalTest inst_repos: daosRepos(),
+                                       inst_rpms: functionalPackages(1, next_version),
+                                       test_function: 'runTestFunctionalV2'
+                    }
+                    post {
+                        always {
+                            functionalTestPostV2()
+                        }
+                    }
+                } // stage('Functional on CentOS 7 with Valgrind')
```

Comment:
Just for organizational purposes, can we put this one after the regular EL FTest?

Reply:
Sure 👍  06f1993f8653ee43fd38d1bba63f6794937d85df